HW04/alexey_shparun/lib/notification.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class Notification
+  attr_reader :notify
+
+  def initialize(notify)
+    @notify = notify
+  end
+
+  def display
+    puts notify
+  end
```

Comment:
```suggestion
```
You can just return notifications, no need to display them

Reply:
Corrected the code, you can review it)